app/controllers/comment_controller.php

```diff
@@ -0,0 +1,55 @@
+<?php
+class CommentController extends AppController
+{
+	CONST MAX_ITEM_PER_PAGE = 10;
+
```

Comment:
Please avoid using magic numbers po. You may declare it as constants :bow: 
Please apply to all